packages/labs/src/labs.md

```diff
@@ -91,6 +91,16 @@ const renderMenuItem = ({ handleClick, item: film, isActive }: ISelectItemRender
 
 @interface ISelectItemRendererProps
 
+@## Suggest
+
+`Suggest` behaves similarly to `Select`, except it exposes a search input as the point of entry, as opposed to a custom element that shows the search in the popover.
```

Comment:
"exposes a search input as the point of entry" doesn't make a lot of sense here, please rephrase.

more like "except it renders a text input as Popover target instead of arbitrary children" ?